cheroot/server.py

```diff
@@ -614,10 +668,6 @@ def read_trailer_lines(self):
 
             yield line
 
-    def close(self):
-        """Release resources allocated for ``rfile``."""
-        self.rfile.close()
-
 
 class HTTPRequest:
```

Comment:
Are you keeping this temporary while refactoring?

Reply:
Close is provided for in the parent class `RFile`